todo-backend/app/Http/Controllers/TodoController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Todo as Todo;
+
+class TodoController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $todos = Todo::all();
+        $id = $request->user()->id;
+        $filter = [];
+        foreach($todos as $t){
+            if($t->user_id == $id) {
+                array_push($filter, $t);
+            }
+        }
+
+        unset($t);
+        return $filter;
+    }
+
+    public function show(Request $request, $id){
```

Comment:
okej logika, ali sad ako obratis paznju 1 metoda u controlleru obavlja vise stvari
1. autorizaciju (da li korisnik sme da vidi taj todo)
2. dobavljanje samog todo-a
a to nije bas najbolje, pogledaj (a i pokazacu ti sutra) policy-je. mozes ih iskoristiti bas da tu logiku oko autorizacije izbacis na drugo mesto

https://laravel.com/docs/5.8/authorization#writing-policies